src/psyclone/dynamo0p3.py

```diff
@@ -4095,8 +4110,16 @@ def upper_bound_halo_depth(self):
         return self._upper_bound_halo_depth
 
     def _lower_bound_fortran(self):
-        ''' Create the associated fortran code for the type of lower bound '''
-        if not config.DISTRIBUTED_MEMORY and self._lower_bound_name != "start":
+        '''
+        Create the associated fortran code for the type of lower bound.
```

Comment:
consistency in case for '[Ff]ortan' please

Reply:
Done.